docs/docs/development/hardware-integration/boards-shields-keymaps.md

```diff
@@ -6,47 +6,53 @@ title: Boards, Shields, and Keymaps
-- A `${board_name}_defconfig` file that forces specific Kconfig settings that are specific to this hardware configuration. Mostly this is SoC settings around the specific hardware configuration.
-- `${board_name}.dts` which contains all the devicetree definitions, including:
+- A `<board_name>_defconfig` file that forces specific Kconfig settings that are specific to this hardware configuration.
+  These are mostly SoC settings around the specific hardware configuration.
+- `<board_name>.dts` which contains all the devicetree definitions[^1], including but not limited to:
   - An `#include` line that pulls in the specific microprocessor that is used, e.g. `#include <st/f3/stm32f303Xc.dtsi>`.
-  - A [chosen](https://docs.zephyrproject.org/3.5.0/build/dts/intro-syntax-structure.html#aliases-and-chosen-nodes) node named `zmk,kscan` which references the configured KSCAN driver (usually a GPIO matrix)
-  - (Optional) A [chosen](https://docs.zephyrproject.org/3.5.0/build/dts/intro-syntax-structure.html#aliases-and-chosen-nodes) node named `zmk,matrix-transform` that defines the mapping from KSCAN row/column values to the logical key position for the keyboard.
+  - Kscan, matrix transform and physical layout devicetree nodes as described above
+  - A [chosen](https://docs.zephyrproject.org/3.5.0/build/dts/intro-syntax-structure.html#aliases-and-chosen-nodes) node including `zmk,physical-layout` property among others, where each property references the nodes defined in the file.
 - A `board.cmake` file with CMake directives for how to flash to the device.
-- A `${board_name}.keymap` file that includes the default keymap for that keyboard. Users will be able to override this keymap in their user configs.
+- A `<keyboard_name>.keymap` file that includes the default keymap for that keyboard. Users will be able to override this keymap in their user configs.
 
-## Pro Micro Compatible Keyboard
+[^1]:
+    Parts of these files can live in separate `.dtsi` files (typically in the same directory) that are then `#include`d in the files, to reduce duplication or improve organization.
+    For instance, a shared `<keyboard_name>.dtsi` file is used for split keyboards that contains devicetree definitions that are shared across keyboard parts.
 
-![Labelled Pro Micro pins](../../assets/interconnects/pro_micro/pinout.png)
+## Composite Keyboard
 
-For keyboards that require a (usually Pro Micro compatible) add-on board to operate, the ZMK integration pieces are places
-in the _shield_ definition for that keyboard, allowing users to
-swap in different Pro Micro compatible boards (e.g. Proton-C, or nice!nano) and build a firmware the matches their actual
-combination of physical components.
+Keyboards that require a (usually Pro Micro compatible) add-on board to operate are [composite keyboards](../../hardware.mdx#composite), where the ZMK integration pieces are placed in the _shield_ definition for that keyboard.
```

Comment:
```suggestion
Keyboards that require a add-on board to operate are [composite keyboards](../../hardware.mdx#composite), where the ZMK integration pieces are placed in the _shield_ definition for that keyboard.
```
I think there are enough xiao keyboards out there now that this mention of pro micro should be removed.